webpack.config.js

```diff
@@ -17,6 +17,11 @@ const webpack = require('webpack');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const WebpackShellPlugin = require('webpack-shell-plugin');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const sass = require('sass');
+const crypto = require('crypto');
+
+const crypto_orig_createHash = crypto.createHash;
+crypto.createHash = algorithm => crypto_orig_createHash(algorithm == 'md4' ? 'sha256' : algorithm);
```

Comment:
md4 or should it be rather md5? Do you remember why this was needed?

Reply:
This is sadly something that I don't have a grasp on. But in short, webpack uses some hash functions that are no longer provided by the crypto package. A proper fix would be to update webpack, but that solution requires way more effort.

Reply:
I've added some comments to make it clear